indico/modules/events/registration/clone.py

```diff
@@ -150,3 +152,11 @@ def _synchronize_registration_friendly_id(self, new_event):
             .filter(Registration.event_id == new_event.id)
             .scalar() or 0
         )
+
+    def _delete_orphaned_registrations(self, event_id):
+        """Delete registrations whose registration form is deleted."""
```

Comment:
This comment is just explaining what the code already explains, but not the reason for it. I'd rather add something like

```python
# Avoid friendly ID conflicts with registrations in deleted regforms
```

(I'd use a comment instead of a docstring since an internal function doesn't really need *documentation*)